### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by gardar** [Polpetta/packer-plugin-podman#49]
Any chance you could create a new release with the recent changes?

Also perhaps you want to register the plugin as a packer integration? You can take a look at other plugins to see what's required to do so:
https://github.com/mondoohq/packer-plugin-cnspec/pull/153
https://github.com/digitalocean/packer-plugin-digitalocean/pull/116

**Comment by Polpetta** [comment]
Yes, I'd publish a new version, since the last one is eons old. First we need to fix #48, but other than that I don't see why not 👍 
Thanks for sharing the links, I didn't know about that! I'll create an issue and mark as a thing to do 😊

**Comment by gardar** [comment]
How about that release now that #48 has been fixed?

**Comment by Polpetta** [comment]
I'll create it now!

**Comment by Polpetta** [comment]
I've published a 1.0, since I think it is a good idea to bump from the previous releases (new content has been updated and other stuff as well). You can find it here: https://github.com/Polpetta/packer-plugin-podman/releases/tag/v1.0.0

**Comment by gardar** [comment]
Great! But it looks like the assets are missing.
I see that the release ci job ran but it seems that goreleaser failed.

**Comment by Polpetta** [comment]
Yep, it seems like go needs to be at least to `1.23`. I've created an issue here #70
